tools/cpu_utilization.py

```diff
@@ -83,7 +83,8 @@ def main(pid: int, output: str, threads: bool) -> None:
                     out.write(f"   {(c[row].system_time - c[row - 1].system_time) * 100 / time_delta:6.2f}% ")
                 else:
                     out.write("     0.00%      0.00% ")
-            row += 1
+            # Ignoring this because it seems like I could delete it but there's no test so I'm unsure
+            row += 1  # noqa: PLW2901
```

Comment:
ping arvid i guess if you still want to keep it.  https://github.com/Chia-Network/chia-blockchain/commit/f9c4f1f96d9d5b82dbf323b7d0e1375ae3734943#diff-23f8e2684a7db826cb75674e4f932b39692062761937d5fee0df43830796f111R84 fwiw

```suggestion
```

Reply:
Okay yeah Arvid said I could delete it.